packages/hardware-ledger/src/LedgerKeyAgent.ts

```diff
@@ -703,6 +703,7 @@ export class LedgerKeyAgent extends KeyAgentBase {
    * Ledger has certain limitations due to which it cannot sign arbitrary combination of all transaction features.
    * The mode specifies which use-case the user want to use and triggers additional validation on `tx` field.
    */
+  // eslint-disable-next-line complexity
```

Comment:
I'm OK with approving this, however we should either look to remove the rule and use other quality gates to catch overly-complex logic or stick with it and refactor to conform.